src/wallet/wallet_utils.rs

```diff
@@ -96,7 +98,10 @@ pub(crate) fn validate_mnemonic_import(
                 return Err(BridgeCliError::AddressMismatch);
             }
         }
-        Err(e) => return Err(BridgeCliError::MnemonicParseError(e.to_string())),
+        Err(e) => {
+            tracing::error!("Error generating address from mnemonic: {}", e);
+            return Err(BridgeCliError::MnemonicParseError);
```

Comment:
I think we had this one incorrect also before, it just caught my eye.

```suggestion
            return Err(BridgeCliError::AddressGenerationFromMnemonicFailed);
```